SWLOR.Game.Server/Feature/StatusEffectDefinition/RestStatusEffectDefinition.cs

```diff
@@ -52,7 +52,7 @@ private void Rest(StatusEffectBuilder builder)
 
                     var hpAmount = 1 + GetAbilityModifier(AbilityType.Vitality, target) / 2;
                     var stmAmount = 1 + GetAbilityModifier(AbilityType.Perception, target) / 2;
-                    var fpAmount = 1 + GetAbilityModifier(AbilityType.Social, target) / 2;
+                    var fpAmount = 1 + GetAbilityModifier(AbilityType.Willpower, target) / 2;
```

Comment:
Good change. I think I did that back when it was still Charisma. This makes a lot more sense.